src/poetry/core/packages/utils/utils.py

```diff
@@ -32,15 +31,15 @@
 SUPPORTED_EXTENSIONS = ZIP_EXTENSIONS + TAR_EXTENSIONS
 
 try:
-    import bz2  # noqa
+    import bz2  # noqa: F401
 
     SUPPORTED_EXTENSIONS += BZ2_EXTENSIONS
 except ImportError:
     pass
 
 try:
     # Only for Python 3.3+
-    import lzma  # noqa
+    import lzma  # noqa: F401
```

Comment:
is python `<3.3` compat still required here?

if we need it, we should use

```python
if sys.version_info >= (3, 3):
    import lzma
    ....
```

or similar

Reply:
I would try to avoid logic change to the code within this PR, because its only about formatting. I think this is better handled in https://github.com/python-poetry/poetry-core/pull/263 for example.

Reply:
ping @tony

Reply:
`lzma` is optionally built in.  You can compile CPython without lzma 

Could not figure out the flags, Not having the library seems to do it:

```
❯ sudo apt remove liblzma-dev
❯ make distclean
❯ ./configure
❯ make -j8
❯ ./python -V
Python 3.11.0a4+

❯ ./python -c 'import lzma'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/user/projects/c/cpython/Lib/lzma.py", line 27, in <module>
    from _lzma import *
    ^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named '_lzma'
```

Reply:
that makes sense